Fix:
error: implicitly declaring library function 'isalnum' with type 'int (int)' [-Werror,-Wimplicit-function-declaration]
error: implicitly declaring library function 'exit' with type 'void (int) __attribute__((noreturn))' [-Werror,-Wimplicit-function-declaration]
--- ctlwords.c.orig	1999-02-14 17:02:25.000000000 -0600
+++ ctlwords.c	2022-01-14 01:07:32.000000000 -0600
@@ -22,7 +22,9 @@
 
 #endif
 
+#include <ctype.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
 
 main(int argc, char *argv[])
